Sources/LiveViewNativeCore/LiveViewNative.swift

```diff
@@ -279,6 +333,19 @@ public struct AttributeName {
         }
     }
 }
+extension AttributeName: ExpressibleByStringLiteral {
+    /// Creates an attribute name from a string literal.
+    ///
+    /// - Note: This initializer does not parse the string for namespaces. To construct an `AttributeName` with a namespace, use ``init(namespace:name:)``.
+    public init(stringLiteral value: String) {
```

Comment:
This kinda feels like a foot gun, because it means string literals don't correspond to the rendered form of an AttributeName. It would be very easy to accidentally introduce bugs this way if namespaced attributes are used. Is it possible to parse the string literal and raise a fatal error/ panic (whatever the Swift equivalent is) if the literal is invalid? If not, I'd recommend removing this impl.

Reply:
I originally left it out because namespaces are used so infrequently (and I can't imagine ever by the client or users thereof) that I didn't think parsing was worth it. I'm fine parsing it, though, if you think preventing that footgun is more important.

Reply:
I know a recent version of LiveView introduced some namespaced attributes in templates like `:if`, which is technically namespaced, just with an empty namespace. I don't know if those are used in LiveView Native though, so if not, then it's probably fine to keep this, otherwise I think we'll need to make sure we parse those correctly.

Reply:
I believe those are only those are only used by the backend and not emitted into the markup. Regardless, this does parse them.

Reply:
If it's only used on the backend, then we can ignore those. The only other one I know uses namespaced names is some of the `svg` element attributes. In either case though, this doesn't correctly parse them, since it will treat `":if"` as an attribute name with no namespace and a name of `:if`, whereas it should treat it as an attribute name with an empty namespace and a name of `if`. Likewise for something like `xlink:href`, as it will treat the whole thing as the name, when it should be namespaced.

I'm going to go ahead and sign off on this and let you make the final call on how to handle namespaced strings

Reply:
The new implementation does correctly parse them, I mean: https://github.com/liveviewnative/liveview-native-core-swift/blob/70c24a9ff304b5525a7bbb2cd801222dfca28260/Sources/LiveViewNativeCore/LiveViewNative.swift#L320